### PR TITLE
Update pt_BR app translation for r3.9.0

### LINKED ISSUE #2698: Update pt_BR app translation for 3.9.0
Hi @melcon,
We are getting ready for the 3.9.0 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **pt_BR** by **2022-07-22**.

Please note the new location of the translation files (.ts): They are now in src/translation. (Refactoring: move src/res/translation to src/translation #2576)

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-07-22

Suggested commit and PR message:
```
Update pt_BR app translation for 3.9.0

Fixes #2698
```

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by melcon**
Fixes #2698

### CODE REVIEW

File: src/translation/translation_pt_BR.ts [hunk 1]
```diff
@@ -2115,17 +2115,17 @@ Silenciamos seu canal e ativamos &apos;Silenciar-me&apos;. Resolva o problema de
...
+        <translation>Ative o alerta de áudio ao receber uma mensagem de bate-papo e quando um novo cliente ingressar na sessão. Um segundo dispositivo de som pode ser necessário para ouvir os alertas.</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="399"/>
         <source>Audio Alerts check box</source>
-        <translation type="unfinished"></translation>
+        <translation>Caixa de seleção de Alertas de Áudio</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="403"/>
```
Comment (pljones) [inline src/translation/translation_pt_BR.ts:2131]: `<source>ASIO Device Settings</source>` This is the button text being referred to in about three places.

File: src/translation/translation_pt_BR.ts [hunk 2]
```diff
@@ -2040,7 +2040,7 @@ Silenciamos seu canal e ativamos &apos;Silenciar-me&apos;. Resolva o problema de
...
-        <translation type="unfinished"></translation>
+        <translation>Alguns drivers de placa de som não permitem que o atraso do buffer seja alterado de dentro do %1. Nesse caso, a configuração de atraso do buffer está desabilitada e deve ser alterada usando o driver da placa de som. Use a ferramenta apropriada para a interface em uso para ajustar o tamanho do buffer. Por exemplo, se estiver usando ASIO, use o botão &quot;ASIO Device Settings&quot; para abrir o painel de configurações do driver ou se estiver usando JACK, use uma ferramenta como QjackCtl para ajustar o tamanho do buffer. Outras interfaces, como o Pipewire, exigem o uso de sua ferramenta apropriada. Consulte o manual da interface.</translation>
```
Comment (pljones) [inline src/translation/translation_pt_BR.ts:2043]: `ASIO Device Settings` in your translation should match the value for `<location filename="../clientsettingsdlg.cpp" line="403"/>` (line 2131, also commented).

File: src/translation/translation_pt_BR.ts [hunk 3]
```diff
@@ -4259,12 +4255,12 @@ Silenciamos seu canal e ativamos &apos;Silenciar-me&apos;. Resolva o problema de
     <message>
         <location filename="../serverdlgbase.ui" line="274"/>
         <source>Custom Directory address</source>
-        <translation type="unfinished"></translation>
+        <translation>End. de Diretório Personalizado</translation>
```
Comment (pljones) [inline src/translation/translation_pt_BR.ts:4258]: Oooh, Google Translate understood that :).

**Comment by ann0see** [comment]
@melcon any updated here?

**Comment by ann0see** [comment]
Also pasting the review checklist here. Please check as apropriate

- [ ] Translator listed in the `src/util.cpp` *optionally add link to PR or code*
- [ ] Punctuation and spacing consistent
- [ ] Signal words consistent ("ASIO", "Buffer")
- [ ] App translations: No untranslated strings (`grep unfinished -5 src/translation/translation_$TRANSLATION*.ts`)
- [ ] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)

File: src/translation/translation_pt_BR.ts [hunk 4]
```diff
@@ -3559,113 +3559,113 @@ Silenciamos seu canal e ativamos &apos;Silenciar-me&apos;. Resolva o problema de
...
         <location filename="../serverdlg.cpp" line="81"/>
         <source>When a value other than &quot;%1&quot; is chosen for Directory, this will show whether registration is successful. If the registration failed, please choose a different directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Quando um valordirefente de &quot;%1&quot; for escolhido para Diretório, isso mostrará se o registro foi bem sucedido. Se o registro falhou, escolha um diretório diferente.</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="134"/>
         <source>No recording directory has been set or the value is not useable. Check the value in the Options tab.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nenhum diretório de gravação foi definido ou o valor não pode ser usado.Verifique o valor na guia Opções.</translation>
```
Comment (ann0see) [inline src/translation/translation_pt_BR.ts:3602]: Space missing after usado

Comment (melcon) [reply src/translation/translation_pt_BR.ts]: done

**Review by pljones — APPROVED** [review]

**Pull request merged**